doc/apiv3-documentation.apib

```diff
@@ -1474,6 +1478,94 @@ This is an example of how a form might look like. Note that this endpoint does n
                 "message": "The resource you are about to edit was changed in the meantime."
             }
 
+# Group MyPreferences
```

Comment:
In the same spirit as not writing "current user", I would rename the type of this resource to `UserPreferences`.

This is basically only forward thinking, in case we will add support for admins seeing/setting user preferences of foreign users some day.

Just to avoid confusion:
I still agree that `/api/v3/my_preferences` is a good URL to point to the own `UserPreferences`, I just don't think that the "my" should carry over into the "class name".